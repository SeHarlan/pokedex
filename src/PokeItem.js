import React, { Component } from 'react';

export default class App extends Component {
    render() {
      return <li className="poke-item">
          <h2>{this.props.item.pokemon}</h2>
          <div>
            <span className="type" style={{backgroundColor:this.props.item.color_1}}>{this.props.item.type_1}</span>
            <span>HP: {this.props.item.hp}</span>
          </div>
          <img src={this.props.item.url_image} alt={this.props.item.pokemon} />
          <div>
            <span>A: {this.props.item.attack}</span>
            <span>D: {this.props.item.defense}</span>
          </div>
      </li>
    }
  }