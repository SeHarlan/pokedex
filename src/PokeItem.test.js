import React from "react";
import renderer from "react-test-renderer";
import PokeItem from './PokeItem.js';

test('renders PokeItem.js correctly', () => {
  const tree = renderer
    .create(<PokeItem item={{}}/>)
    .toJSON();
  expect(tree).toMatchSnapshot();
})