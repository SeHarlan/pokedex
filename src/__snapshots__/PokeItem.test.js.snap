// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders PokeItem.js correctly 1`] = `
<li
  className="poke-item"
  style={
    Object {
      "borderColor": undefined,
    }
  }
>
  <h2 />
  <div>
    <span
      className="type"
      style={
        Object {
          "backgroundColor": undefined,
        }
      }
    />
    <span>
      HP: 
    </span>
  </div>
  <img />
  <div>
    <span>
      A: 
    </span>
    <span>
      D: 
    </span>
  </div>
</li>
`;
